package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contentful-migrations-action",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
this version already exists https://github.com/the-infatuation/contentful-action/releases/tag/v2.1.0

Reply:
```suggestion
  "version": "2.1.2",
```

Reply:
😱